CLOUDP-128165: Create command "atlas backups schedule describe" (#1360)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-backups-schedule-describe.txt

.. _atlas-backups-schedule-describe:

===============================
atlas backups schedule describe
===============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Describe a cloud backup schedule of a given cluster.

Reviewer: ```suggestion
Describe a cloud backup schedule for the cluster you specify.
```

Syntax
------

.. code-block::

   atlas backups schedule describe <clusterName> [options]

Arguments
---------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - clusterName
     - string
     - false

>      - Name of the Cluster.

Reviewer: ```suggestion
     - Human-readable label for the cluster.
```
